test/test_cuda.py

```diff
@@ -1509,6 +1509,31 @@ def test_events(self):
         self.assertTrue(event.query())
         self.assertGreater(start_event.elapsed_time(event), 0)
 
+    def _test_event_handle_consumer(handle):
+        e1 = torch.cuda.Event(_handle=handle)
+        # synchronization here is not really necessary, as the torch.cuda.Event
+        # invocation above will block until the current device wakes up. This is
```

Comment:
@colesbury is this behavior expected?

Reply:
it is actually the process spawn invocation that blocks

Reply:
1. call `Event.ipc_handle()` on a different device hits segfault. 
2. if create the handle on the same device then pass it to a different device, the subprocess spawning will also block until the first device wakes up.